test/README

```diff
@@ -1,45 +1,75 @@
-# Automated Testing of Cockpit
+# Integration Tests of Cockpit
 
-This directory contains automated tests for Cockpit, and the support
+This directory contains automated integraton tests for Cockpit, and the support
 files for them.
 
-To run the tests, you need to install the following packages on
-Fedora:
+To run the tests on Fedora, you need to install the following packages:
 
-  # yum install npm python-libguestfs qemu mock qemu-kvm python \
-         curl libvirt-client openssl rpmdevtools libguestfs-tools \
-         rpm-build krb5-workstation python-lxml expect rsync \
-         selinux-policy-devel xz
+    $ sudo yum install npm python-libguestfs qemu mock qemu-kvm python \
+         libvirt-client openssl libguestfs-tools selinux-policy-devel \
+         rpm-build krb5-workstation expect rsync libvirt-python curl xz
 
 Testing requires phantomjs globally on the system, not just inside the cockpit package.
 
-  # npm -g install phantomjs
+    $ sudo npm -g install phantomjs
 
-## Quick list of tools
+## Introduction
+
+To run the integration tests run the following. Don't run the integration tests
+as root:
+
+    $ ./VERIFY
+
+The first time you run the tests, they will create a cockpit1 bridged network used
+for testing. You may be required to type your sudo password for this to work. In
+addition various test images will be downloaded. This can take some time.
+
+Alternatively you can run an individual test like this:
+
+    $ cd test
+    $ ./testsuite-prepare
+    $ ./check-session
+
+The `./testsuite-prepare` command builds cockpit and puts it into the test. Only
+code currently committed in the current branch will be installed and tested.
+
+To see more verbose output from the test, use the --verbose and/or --trace flags:
+
+    $ ./check-session --verbose --trage
+
+In addition if you specify `--sit`, then the test wail wait on failure and allow
+you to log into cockpit and/or the test instance and diagnose the issue. An address
+will be printed of the test instance.
+
+    $ ./check-session --trace --sit
+
+## Details
 
 For setting up the host:
 
-vm-prep,     for creating the test network etc.
+ * vm-prep: Creates the test network etc.
 
-For managing test machines:
+For managing test machines and their images:
 
-vm-create,   for creating (or downloading) test machine images.
-vm-reset,    for starting over from a clean slate.
-vm-install,  for installing RPMs into a test machine image.
+ * vm-download: Downloads test images
```

Comment:
for all of these tools, I think it might be nicer to read if we use the imperative form, not third person
e.g.: "Download test images"